cmd/provisioner/hostpath-provisioner.go

```diff
@@ -91,11 +96,11 @@ func NewHostPathProvisioner() controller.Provisioner {
 
 var _ controller.Provisioner = &hostPathProvisioner{}
 
-func isCorrectNodeByBindingMode(annotations map[string]string, nodeName string, bindingMode storage.VolumeBindingMode) bool {
+func isCorrectNodeByBindingMode(annotations map[string]string, nodeName string, provisionerName string, bindingMode storage.VolumeBindingMode) bool {
 	glog.Infof("isCorrectNodeByBindingMode mode: %s", string(bindingMode))
 	if _, ok := annotations["kubevirt.io/provisionOnNode"]; ok {
 		if isCorrectNode(annotations, nodeName, "kubevirt.io/provisionOnNode") {
-			annotations[annStorageProvisioner] = defaultProvisionerName
```

Comment:
Not very sure what the purpose of this line is, appreciate if someone could enlighten me.